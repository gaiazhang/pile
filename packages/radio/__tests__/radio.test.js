import React from 'react';
import { shallow } from 'enzyme';
import Radio from '../src';

describe('Radio suite', () => {
  it('render a radio', () => {
    const wrapper = shallow(<Radio />);
    expect(wrapper).toMatchSnapshot();
  });
});
