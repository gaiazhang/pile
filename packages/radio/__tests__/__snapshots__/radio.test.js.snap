// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Radio suite render a radio 1`] = `
<withState(withHandlers(Component))
  prefixCls="pile"
  size={null}
  updateValue={[Function]}
  value=""
/>
`;
